lib/src/Base/Stat/CovarianceModelImplementation.cxx

```diff
@@ -118,6 +121,7 @@ CovarianceModelImplementation::CovarianceModelImplementation(const Point & scale
   , outputCovarianceCholeskyFactor_(0)
   , definesComputeStandardRepresentative_(false)
   , isDiagonal_(true)
+  , isStationary_(false)
```

Comment:
now with c++11 you can write "Bool isStationary_ = false" in the .hxx and you're done

Reply:
yep thanks. But the idea is to have similarities with other attributes